### PR TITLE
feat [SCRUM-87]: added more dropdown options, tab switcher revised and language refined

### PR DESCRIPTION
**Pull request by JxnLexn**
# Description

Added all required dropdown options, tab switcher revised and language refined

## ⁉️ Type Of Change

- [x] ⚙️ New feature (non-breaking change which adds functionality).
- [ ] 🪛 Bug fix (non-breaking change).
- [ ] ⚠️ Breaking change (fix of feature that would cause existing functionality to not work as expected).
- [ ] ✏️ This change requires a documentation update.

## 🧪 How Has This Been Tested?

Dummy GUI app.py executed

## ✅ Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by ki1yoshi** [comment]
Top

**Review by ki1yoshi — DISMISSED** [review]
Top

File: frontend/mainGui.ui
Comment (PI860) [inline frontend/mainGui.ui:1]: Z479 Samples deutsch

Comment (PI860) [inline frontend/mainGui.ui:1]: Z702 Result deutsch

Comment (PI860) [inline frontend/mainGui.ui:1]: Z770 Result deutsch

**Review by PI860 — COMMENTED** [review]
Bitte die 3 Sachen noch auf deutsch ändern

File: frontend/mainGui.ui
Comment (JxnLexn) [reply frontend/mainGui.ui:1]: Ist schon deutsch

Comment (JxnLexn) [reply frontend/mainGui.ui:1]: Korrigiert

Comment (JxnLexn) [reply frontend/mainGui.ui:1]: Korrigiert

**Review by ki1yoshi — APPROVED** [review]
Sieht super aus, außer Paula findet noch was

**Review by PI860 — APPROVED** [review]
Bassd scho :)

**Pull request merged**